specification/ai/Azure.AI.Projects/agents/vector_stores/common/main.tsp

```diff
@@ -87,3 +87,46 @@ model VectorStoreStaticChunkingStrategyResponse
   /** The options for the static chunking strategy. */
   static: VectorStoreStaticChunkingStrategyOptions;
 }
+
+/** Type of vector storage configuration. */
+union VectorStorageDataSourceAssetType {
```

Comment:
please rename to VectorStoreDataSourceAssetType

Reply:
Fixed